build.gradle

```diff
@@ -55,7 +57,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0-beta03'
+        classpath 'com.android.tools.build:gradle:3.3.0-alpha03'
```

Comment:
should we be using an alpha of the build tools? was there a reason for this change?

Reply:
Nope. It was accidental, removed it.